packages/nifti-volume-loader/src/helpers/scaleNiftiArray.ts

```diff
@@ -0,0 +1,23 @@
+/**
+ * Given a pixel array, rescale the pixel values using the rescale slope and
+ * intercept and if modality is PT it uses the suv values to scale the array
```

Comment:
I do not see in the code where the check for PT is, nor the SUV scaling. What am I missing?